src/encoded/tests/fixtures/schemas/biosample.py

```diff
@@ -375,4 +375,3 @@ def biosample_23(testapp, lab, award, source, organism, epiblast):
         'source': source['uuid']
     }
     return testapp.post_json('/biosample', item, status=201).json['@graph'][0]
```

Comment:
please add one new line in the of the file

Reply:
New line is there